core/hydromet_stratified.py

```diff
@@ -0,0 +1,560 @@
+import pathlib as pl
+import numpy as np
+import pandas as pd
+from scipy.integrate import quad
+from scipy.optimize import minimize
+from scipy import interpolate, stats, special
+from matplotlib import pyplot as plt
+from cycler import cycler
+
```

Comment:
You might consider putting some info here describing the scope of this file.

Reply:
Added as issue #53.